_conferences/droidcon-beijing-2017.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "Droidcon"
+website: http://droidcon.cfict.com.cn/
+location: Beijing, China
+
+date_start: 2017-11-17
+date_end:   2017-11-18
+
+cfp_site: http://droidcon.cfict.com.cn/828378719/index?pageId=828448831
```

Comment:
You need to add a cfp_start and cfp_end for this to show up

Reply:
@JakeWharton I am not sure about `cfp_start` and `cfp_end` here. Should I just remove the `cfp_site` in that case?

Reply:
I'd set `cfp_start` to today, `cfp_end` to `date_start` with a comment that the real `cfp_end` date is unknown. That way the CfP link will show up.
Bonus points if you contact the organizers to ask them to update their site with a CfP deadline.

Reply:
confirmed with organizers via email, added `cfp_end`